binding/tfjs_binding.cc

```diff
@@ -125,7 +125,9 @@ static napi_value ExecuteOp(napi_env env, napi_callback_info info) {
 
 static napi_value InitTFNodeJSBinding(napi_env env, napi_value exports) {
   napi_status nstatus;
-  gBackend = new TFJSBackend(env);  // Is cleanup available?
+
+  gBackend = TFJSBackend::Create(env).release();
```

Comment:
This doesn't make sense to use `std::unique_ptr` now - let's just return a naked pointer instead.